features/graph/api/src/main/java/org/opennms/netmgt/graph/api/generic/GenericEdge.java

```diff
@@ -62,6 +64,13 @@ public void setProperty(String key, Object value) {
         properties.put(key, value);
     }
 
+    @Override
```

Comment:
@mvrueden I had to add this to not end in an endless recursion for the toString() method

Reply:
Yeah, that is a known bug. Please add a TODO MVR or TODO PS in order to revert these, when it is fixed.

Reply:
done